linode/manager
# M3-5384: Add support for multiple payment methods in make payment drawer

## Description
- Adds support for displaying multiple payment methods in the Make a Payment drawer
- If a user is making a payment through the action dropdown, auto-select that payment in the Make a Payment drawer
- Refactor CardBase into a functional component
- Refactor variant logic out of SelectionCard; accept a renderVariant function instead, remove duplicate css
    - I went ahead and deleted the check mark since it was only being used in one other place and selectable does the job

## How to test
The Make a Payment drawer should now display multiple payment methods. The default payment method is auto-selected unless the user opened the drawer through the payment method dropdown menu, then that payment method is selected. Making a payment should charge the selected card (check the network tab).

Check places in the app where the SelectionCard component is used, ensure it looks good on mobile and that there are no regressions.
